tests/test_minimizer.py

```diff
@@ -197,3 +199,44 @@ def test_local_minimize_water_box():
     u_init_test, g_init_test = val_and_grad_fn(x0)
     assert u_init == u_init_test
     np.testing.assert_array_equal(g_init, g_init_test)
+
+
+@pytest.mark.xfail(reason="Doesn't work currently, but should eventually")
+def test_minimizer_failure_toy_system():
+    """https://github.com/proteneer/timemachine/pull/1346 introduced a minimization scheme that minimizes
+    at lambda = 0.1 and at 0.0. The need for lambda = 0.1 was due to minimizing PFKFB3 with ligands
+    20 and 43, this test replicates the failure case with a small toy system to allow investigation in the future
+    """
+
+    # Atoms 2933 and 38815 from PFKFB3 with ligands 20, 43
+    nb_params = np.array(
+        [
+            [0.9323587, 0.13247664, 0.25629826, 0.0],
+            [0.52629348, 0.12916129, 0.26202344, 0.0],
```

Comment:
I was incorrect about the eps being zero. Realized I had been looking at the padded values of the NonbondedAllpairs rather than the ligand-protein ixn group.

Reply:
> I was incorrect about the eps being zero.

Unsure of the context?